.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,31 @@
+<!-- Provide a brief description of the PR, if the title is insufficient -->
+
+#### Related PRs / Issues
+
+<!-- List any related issues here -->
+
+#### UI / API Impact
+
+<!-- Roughly, how would this affect the current API or user-facing interfaces? (extend, deprecate, remove, or break) -->
+<!-- Of note: manager config.ini interface, targetutils & bridge scala API, platform config behavior -->
+
+#### Verilog / AGFI Compatability
+
+<!-- Does this change the generated Verilog or the simulator memory map of the default targets?  -->
+
+### Contributor Checklist
+
+- [ ] Did you add Scaladoc to every public function/method?
```

Comment:
This should probably be expanded to say Scaladoc/doxygen/docstrings to every public member (in Scala/C++/Python)?  But starting with Scaladoc is great.